cves/kernel/CVE-2016-2547.yml

```diff
@@ -364,8 +369,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: check if all instances are locked
```

Comment:
In general, notes like this should be more complete to add confidence in your work